products/opensuse/main.pm

```diff
@@ -939,6 +939,11 @@ elsif (get_var('INSTALL_LTP')) {
     loadtest 'kernel/boot_ltp';
     loadtest 'kernel/shutdown_ltp';
 }
+elsif (get_var('LTP_SETUP_NETWORKING')) {
```

Comment:
Well the setup **is** quite complicated, that's why (there are going to be more packages installed, services configured and started in the future). I wouldn't repeatedly run it before each test. And also it can break things, so that's why I separated it.
Can it stay how it is?